ifrs17-template/Test/scenarioYieldCurve.csv

```diff
@@ -0,0 +1,9 @@
+@@Main
```

Comment:
for the file name, use the same convention we have for the other files:
YieldCurve_2020_12_YCUP1.csv

Reply:
done